lib/Pill/common.js

```diff
@@ -20,5 +20,6 @@ export const types = {
   default: 'default',
   red: 'red',
   green: 'green',
-  purple: 'purple'
+  purple: 'purple',
+  greenPrimary: 'greenPrimary'
```

Comment:
We could probably name this better to try and make the two greens a bit more descriptive. `greenWhiteText` or something!